src/components/Sections/SectionChart/SectionBarChart.less

```diff
@@ -11,8 +11,9 @@
 }
 .section-column-chart {
   height: 100%;
+  padding-top: 5px;
```

Comment:
why would you add padding and negative top?

Reply:
cause we have a white background , 
i need to move up the legend so it won't overlap the top axis value.